cmd/osd/main.go

```diff
@@ -164,6 +165,10 @@ func main() {
 			Name:  "jwt-ecds-pubkey-file",
 			Usage: "JSON Web Token ECDS Public file path",
 		},
+		cli.StringFlag{
+			Name:  "jwt-system-shared-secret",
+			Usage: "JSON Web Token system shared secret",
```

Comment:
I know what this is, but other users could use a little more information. Something like "This shared secret is used by clusters to create tokens to communicate with eachother", something like that.